WebInterface/app/controllers/graph_controller.rb

```diff
@@ -0,0 +1,64 @@
+require('net/http')
+
+class GraphController < ApplicationController
+  def show
+    setting = Setting.find(params[:id])
+    actuals = get_actuals(setting.start_date, setting.end_date)
+    unless actuals == nil
```

Comment:
Det er ikke så vigtigt, men der er den her methode:

```suggestion
    unless actuals.nil?
```